qemu/tests/virtio_fs_share_data.py

```diff
@@ -811,6 +810,11 @@ def check_security_label(file, folder, xattr_name):
         if os_type == "windows":
             win_driver_utils.memory_leak_check(vm, test, params)
     finally:
+        if os_type == 'windows' and vm and vm.is_alive():
```

Comment:
Hi @xiagao 
I think just this part relates to the commit message, but other changes do not. 
So I don't understand the purpose of those changes.
Could you split those changes into different commits?

Reply:
Hi @YongxueHong 
I update the commit, for the other code, most of them are parameters update because we use some unit function in https://github.com/autotest/tp-qemu/pull/3596, some parameters are inconsistent.